python/dataset.cpp

```diff
@@ -225,18 +254,18 @@ void init_dataset(py::module &m) {
 
   bind_helper_view<items_view, Dataset>(m, "Dataset");
   bind_helper_view<items_view, DatasetView>(m, "DatasetView");
-  bind_helper_view<items_view, CoordsView>(m, "CoordsView");
+  bind_helper_view<str_items_view, CoordsView>(m, "CoordsView");
   bind_helper_view<items_view, MasksView>(m, "MasksView");
   bind_helper_view<keys_view, Dataset>(m, "Dataset");
   bind_helper_view<keys_view, DatasetView>(m, "DatasetView");
-  bind_helper_view<keys_view, CoordsView>(m, "CoordsView");
+  bind_helper_view<str_keys_view, CoordsView>(m, "CoordsView");
   bind_helper_view<keys_view, MasksView>(m, "MasksView");
   bind_helper_view<values_view, Dataset>(m, "Dataset");
   bind_helper_view<values_view, DatasetView>(m, "DatasetView");
   bind_helper_view<values_view, CoordsView>(m, "CoordsView");
```

Comment:
Should this have been changes as well to `str_keys_view`?

Reply:
I don't think this needs to be changed as the values view is used to returns the vales of the coord dict which do not need to be converted.